lib/spacex.rb

```diff
@@ -20,3 +20,5 @@
 require_relative 'spacex/rockets'
 require_relative 'spacex/ships'
 require_relative 'spacex/version'
+
+require 'spacex/models'
```

Comment:
What is the reason for having `require` instead of `require_relative` ?

Reply:
I can't think of a good reason. Will update.